src/hotspot/share/gc/g1/g1ConcurrentMark.cpp

```diff
@@ -2090,6 +2079,23 @@ void G1ConcurrentMark::abort_marking_threads() {
   _second_overflow_barrier_sync.abort();
 }
 
+double G1ConcurrentMark::worker_threads_cpu_time_s() {
+  class CountCpuTimeThreadClosure : public ThreadClosure {
+  public:
+    jlong _total_cpu_time;
+
+    CountCpuTimeThreadClosure() : ThreadClosure(), _total_cpu_time(0) { }
+
+    void do_thread(Thread* t) {
+      _total_cpu_time = os::thread_cpu_time(t);
```

Comment:
Shouldn't this be `+=` rather than just `+`?

Reply:
nice catch! Fixed.